packages/web-twig/src/Resources/components/Modal/README.md

```diff
@@ -1,7 +1,187 @@
 # Modal
 
+## Composed Modal (DEPRECATED)
+
 This is Twig implementation of the [Modal] component.
 
+Modal is a composition of several subcomponents:
+
+- [ModalComposed](#modalcomposed)
+  - [ModalHeader](#modalheader)
+  - [ModalBody](#modalfooter)
+  - [ModalFooter](#modalfooter)
+
+⚠️ **DEPRECATED:** The component `ModalComposed` — which is currently necessary to
+distinguish the composed version of Modal from the simple one — is deprecated
+and will be removed in next major version: the composed Modal will supersede the
+[Simple Modal](#simple-modal-deprecated) variant.
+
+### ModalComposed
+
+#### Examples:
+
+```twig
+<ModalComposed id="modal-composed">
+  …
+</ModalComposed>
+<ModalComposed id="modal-composed" isExpandedOnMobile titleId="modal-composed-title">
+  …
+</ModalComposed>
+```
+
+👉 Please note the `titleId` attribute is linked to the title inside
+[Modal Header](#modalheader) and provides an accessible name for the dialog.
+
+#### Expand on Mobile Screens
+
+We recommend to expand the Modal on mobile screens using the
+`isExpandedOnMobile` prop. If you omit the prop, the Modal shrinks
+to fit the height of its content (if smaller than viewport).
```

Comment:
The recommendation is meant for the `web-twig` package. We can make it default in Twig and React.